package.json

```diff
@@ -37,7 +37,7 @@
         "lint-staged": "npx lint-staged",
         "lint:shellcheck": "./scripts/shellcheck.sh",
         "_______ Global Scripts _______": "Shared scripts for running in all workspaces",
-        "g:eslint": "cd $INIT_CWD && eslint --report-unused-disable-directives --cache --ignore-path ../../.gitignore",
+        "g:eslint": "cd $INIT_CWD && eslint --report-unused-disable-directives --ignore-path ../../.gitignore",
```

Comment:
This is a mistake, already fixed it. Will be part of a fixup commit. But good catch. That would definitely affect CI performance.

Reply:
Not really this could affect only local performance, there two caches one in Nx which will work independently and second in eslint which is one you disabled. Eslint cache is not used on CI because files for this cache is gitignored. But yea, definitely better to keep this on :D